src/routes/routes.ts

```diff
@@ -1,23 +1,25 @@
 import { Router, Request, Response } from 'express'
 import { createBooking } from '../controllers/book'
+import { handleLogin } from '../controllers/login'
+import { authenticate } from '../middlewares/auth'
+import bodyParser from 'body-parser'
+import { getBookings } from '../controllers/bookings'
+import { editBooking } from '../controllers/edit'
+import { deleteBooking } from '../controllers/delete'
 
 export const router: Router = Router()
 
 // landing page
-router.get('/', (req: Request, res: Response) => {
+router.get('/', authenticate, (req: Request, res: Response) => {
```

Comment:
actual route for landing page shouldn't require auth

Reply:
Ok, updated routes for those that should be public